src/jbi/bugzilla.py

```diff
@@ -41,6 +41,18 @@ class BugzillaWebhookEvent(BaseModel):
     target: Optional[str]
     routing_key: Optional[str]
 
+    def changed_fields(self) -> Optional[List[str]]:
+        """Returns the names of changed fields in a bug"""
+        if self.changes:
+            return [c.field for c in self.changes]
+
+        # Private bugs don't include the changes field in the event, but the
+        # field names are in the routing key.
+        if self.routing_key is not None and self.routing_key[0:11] == "bug.modify:":
+            return self.routing_key[11:].split(",")
+
+        return None
```

Comment:
It differentiates between the case where the event is a change event (bug.modify) or a create event (bug.create) where changes is sort of nonsensical. I don't know that it matters much in practice since if you care about the difference you can look at `event.action` so I can change it if you like.